zebra/debug_nl.c

```diff
@@ -576,6 +577,86 @@ const char *nhm_rta2str(int type)
 	}
 }
 
+const char *frh_rta2str(int type)
+{
+	switch (type) {
+	case FRA_DST:
+		return "DESTINATION ADDRESS";
```

Comment:
These should match what other 2str's are doing and just drop the the FRA half.

Reply:
done